### ISSUE TITLE
cannot reproduce RACE accuracy

### ISSUE THREAD

**Issue by TianrenWang** [zihangdai/xlnet#232]
I am using Google Cloud's TPU to reproduce the findings for RACE. I am using the script with batch_size=8, but after 4000 steps of training with the original setting I got the following evaluation performance:

```
loss 1.3862953186 | eval_accuracy 0.255371391773 | eval_loss 1.38626658916 | global_step 4000
```

I reduced the learning rate from 2e-5 to 1e-6, and I got the following evaluation performance:

```
eval_accuracy 0.539390206337 | eval_loss 1.4650387764 | global_step 12000
eval_accuracy 0.612031936646 | eval_loss 1.37905335426 | global_step 24000
```

What settings did you use to reach the 80% accuracy as described in the README?
